src/RestApi/Controllers/Products.php

```diff
@@ -350,13 +350,16 @@ protected function get_images( $product ) {
 			}
 
 			$thumbnail = wp_get_attachment_image_src( $attachment_id, 'woocommerce_thumbnail' );
+			$metadata  = wp_get_attachment_metadata( $attachment_id, 'full' );
```

Comment:
Can you explain why we need this? I saw some usage above for aspect ratio but how do you know the srcset/size being shown matches the aspect ratio of the full image? What about cropping?